ush/python/pygfs/task/upp.py

```diff
@@ -259,6 +259,6 @@ def finalize(upp_run: Dict, upp_yaml: Dict) -> None:
             Fully resolved upp.yaml dictionary
         """
 
-        # Copy "upp_run" specific generated data to COM/ directory
-        logger.info(f"Copy '{upp_run}' processed data to COM/ directory")
+        # Copy "upp_run" specific generated data to COMIN/ or COMOUT/ directory
+        logger.info(f"Copy '{upp_run}' processed data to COMIN/ or COMOUT/ directory")
```

Comment:
```suggestion
        # Copy "upp_run" specific generated data to COMOUT/ directory
        logger.info(f"Copy '{upp_run}' processed data to COMOUT/ directory")
```

Reply:
@WalterKolczynski-NOAA 
I agree with your suggestion and make the changes.

Reply:
@WalterKolczynski-NOAA 
Yes, I have made changes based on your suggestion.